src/awx_plugins/interfaces/_temporary_private_container_api.py

```diff
@@ -39,7 +40,7 @@ def get_incontainer_path(
     if not os.path.isabs(private_data_dir):
         raise RuntimeError('The private_data_dir path must be absolute')
 
-    container_root = pathlib.Path(CONTAINER_ROOT)
+    container_root_path = pathlib.Path(container_root or CONTAINER_ROOT)
```

Comment:
(don't forget to adjust variable name in the return statement)

Reply:
fixed it up